typescript/sdk/src/core/HyperlaneCoreDeployer.ts

```diff
@@ -133,7 +135,9 @@ export class HyperlaneCoreDeployer extends HyperlaneDeployer<
         !e.message.includes('Reverted 0x08c379a') &&
         // Handle situation where the gas estimation fails on the call function,
         // then the real error reason is not available in `e.message`, but rather in `e.error.reason`
-        !e.error?.reason?.includes('already initialized')
+        !e.error?.reason?.includes('already initialized') &&
+        // Some providers, like on Viction, return a generic error message for all revert reasons
+        !e.message.includes('always failing transaction')
```

Comment:
bruh

Reply:
thank god we are getting rid of this try/catch stuff with CRUD :)

Reply:
hahaha very excited for that